lib/cmds/login.ts

```diff
@@ -55,28 +54,25 @@ export const login = async ({
     token = managementTokenFlag
   } else {
     if (managementToken) {
-      log()
-      log(
-        `Looks like you already stored a management token on your system. ${chalk.dim(
-          `(Located at ${await getConfigPath()})`
-        )}`
+      console.log(
+        `You're already logged in! to logout type: ${chalk.green(
```

Comment:
Personally I prefer the old message with the "maybe you want to log out?" coming later, I think it's a good separation whereas the logout hint is a bit harder to spot here.